### DISCUSSION TITLE
List of medical balance issues

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#12497] [Ideas and suggestions]
There are a few aspects of Medical in barotrauma that could be improved without fundamentally changing the medical system
I'm preparing a PR with some changes to try and address those

1. Many useless meds
2. Lack of "skill"
3. Poisons / Offensive meds

# Useless meds

Counting curative medicines (remove afflictions), there are 26 meds in the game. 
Some of them are never used for a variety of reasons:

- **Antibiotic Glue:** Massively heals a SINGLE limb. 3 bandages can do the same for a much cheaper cost.
- **Liquid Oxygenite**: CPR is too good  #12468, therefore injecting someone with oxygen is pointless
- **Fentanyl**: Expensive and much higher medical skill requirement than Morphine
- **Saline**: Heals less than the much more abundant, risk-free alien blood 
- **Antidotes**: Stabilozine spam exists, antidotes are expensive and redundant. See Poisons
- **Adrenaline**: Doesnt do anything, only used in Fentanyl

On the other hand, some meds are too good:

- **Alien Blood**: is EVERYWHERE, heals a lot and has no drawbacks. Bleeding out in a fight outside isnt an issue when you can loot a random crawler and patch yourself up
- **Morphine:** does everything. A general purpose healing items isnt a problem, but morphine does it all too well, too easily (see "Lack of skill")

# Lack of Skill

The medical system lacks depth, and you cannot become a "good" medic besides gaining the medical skill stat. 
Outside of the stat, medical "skill" comes from 

- **Morphine does everything**
Morphine cures all of the common afflictions that aren't blood loss. It also partially cures burns on the whole body, something reserved for bandages. The issue is that it does that too easily with little drawback. See Below

- **Limb-specific is underused.** 
Bandages are the only real limb-specific medicine, but usually applying by clicking each limb and spamming the hotkey. Other afflictions are either not limb specific, or healed by medicine that isn't limb specific.

- **The side effects (mostly) don't matter.** 
Morphine for example has 3 side effects: Low O2, Addiction and Overdose, and yet you can be reckless with it.
For Addiction/Withdrawal negative effects to kick in, you need to fail 2 Morphine checks, a 10% or chance of failing 2 morphines on spawn stats. Low O2 side effect doesn't matter due to how strong CPR is.
Alien Blood's side effect is psychosis, which doesn't impact the player's health
Stabilozine has no side effect (see poisons). 

# Poisons / Offensive meds

Offensive meds, including Poisons but also non lethal options like Chloral Hydrate are useless

- **Offensive meds kill slower than bullets**. Poisons/Acid will always compete against bullets for killing enemies, and both are most of the time slower than bullets, while also not inflicting stun. 
Medic talents "What a stench" and "Macrodosing" also inflate the damage numbers of poisons/acid, making sure none but the medic who specialized can use them effectively. 

- **No enemy uses poisons.** Also Cyanide/Morbusine kills too quickly to be used in that case, leaving only Sufforin

- **Stabilozine spam makes antidotes useless**. Stabilozine is cheap and doesnt have any side effects. Antidotes are only used to cure the (non existent) poisons and contribute to item bloat. Also stabilozine is 5x more effective on Morbusine and Sufforine than on Cyanide due to how fast it ramps up

- **There is no reason to use non-lethals.** No mission objectives ask you to incapacitate enemies, especially fishes. Chloral Hydrate or Paralyzant are therefore useless.

**Comment by Ek-Videogames** [comment]
> Offensive meds kill slower than bullets. Poisons/Acid will always compete against bullets for killing enemies, and both are most of the time slower than bullets, while also not inflicting stun.

Not to mention lots of enemies are just entirely immune to poisons because of: 
conscious balance decisions, intentional/unintentional 'vitality scaling' quirks, or massive amounts of HP relative to a human.

**Comment by NANO-Dragoon** [comment]
Isn't the fact that alien blood give psychosis a drawback? I mean I feel like it should give more like maybe the amount of psychosis it gives matches the amount of blood it gives? But I still don't think it's a pointless drawback considering too much of it, can make you see and hear things. As well as make stuff invisible, Using a bunch of it mid battle would be bad. I don't think it needs to harm the players health.

I do agree with most of the rest. I think enemies using poisons would be interesting. Not only would it make cures more used, but it would also make it possible for players to get more poisons as well! And you are 100% correct about the non-lethals.

**Comment by bookslayer10** [comment]
Non-lethals are very useful when you need to take or keep people prisoner. Chloral Hydrate could stand to have its duration buffed, but it's already powerful in PvP where you can just stun someone through armor and take all their things. Paralyzant is very useful when transporting prisoners, just paralyze them and store them somewhere they won't drown.

I agree with everything else though, except saline, hallucinations are a downside that makes me just take saline instead. Sure, it's less powerful, but it's so cheap because you get three for every large enemy you kill. Really, the useless item is the blood bag, it's just a waste of resources compared to the difference it makes.

**Reply by Ek-Videogames** [reply]
Not to mention 'blood loss' affliction isn't often gained in large amounts (unless in a gunfight) - since most damage sources don't cause very much bleeding (besides gunshots) and any half-competent person will know to bandage themselves or a downed teammate almost immediately. Blood bags are sort of useless/"overkill" for this same reason, saline works good enough in most common situations where you only have like 20% blood loss to get rid of.

Honestly for balance maybe it'd be better if chloral hydrate was 'less effective' when fired from syringe gun or swung as a melee weapon - and instead was a more incremental stun like stun batons are, but using chloral hydrate in the medial GUI 'proper' would still give a proper sedative effect. (My rationale being it'd be less instant and need to score multiple direct hits, but if you say; already stunned somebody with a baton and then injected them on the ground to keep them unconscious for longer then it would work as intended.)

**Reply by bookslayer10** [reply]
I like the idea of needing to inject Chloral Hydrate through the health menu to get a stronger effect, but I think that it would be better to just quadruple the effect of the stun if the person is is being injected into is already stunned. Still useful for combat, but if you have already stunned them you can extend the stun long enough to move them around easily.
